web-dev/src/components/Profile/index.js

```diff
@@ -14,6 +14,17 @@ const ProfileScreen = () => {
     // const profile = useSelector(state => state.profile);
     // console.log(JSON.stringify(profile))
 
+    /*
```

Comment:
Here I wrote down the setup idea that I discussed with Georgian. 

When we click on the other User, with the username like mamaboy, it will navigate to the link with format '/profile/?username=mamaboy'.

Then from there we make sure whether mamaboy is also the user in context, if YES, we do nothing since we are already at the main user's profile page. If NOT, we nagivate to 'profile/mamaboy'. This will lead to the mamaboy's public profile.